.cicd/scripts/srw_ftest.sh

```diff
@@ -71,12 +71,27 @@ echo "DATA_LOCATION=${DATA_LOCATION}"
 sed "s|^task_get_extrn_ics:|task_get_extrn_ics:\n  EXTRN_MDL_SOURCE_BASEDIR_ICS: ${DATA_LOCATION}/FV3GFS/grib2/2019061518|1" -i ush/config.yaml
 sed "s|^task_get_extrn_lbcs:|task_get_extrn_lbcs:\n  EXTRN_MDL_SOURCE_BASEDIR_LBCS: ${DATA_LOCATION}/FV3GFS/grib2/2019061518|1" -i ush/config.yaml
 
+hpss_machines=( jet hera )
+
+# Use staged data for HPSS supported machines
+if [[ ${hpss_machines[@]} =~ ${platform,,} ]] ; then
+    sed 's|^task_get_extrn_ics:|task_get_extrn_ics:\n  USE_USER_STAGED_EXTRN_FILES: true|g' ush/config.yaml
+    sed 's|^task_get_extrn_lbcs:|task_get_extrn_lbcs:\n  USE_USER_STAGED_EXTRN_FILES: true|g' ush/config.yaml
+fi
+
 # Activate the workflow environment ...
 source etc/lmod-setup.sh ${platform,,}
 module use modulefiles
 module load build_${platform,,}_${SRW_COMPILER}
 module load wflow_${platform,,}
 
+# Load more modules on machines with hpss access
+if [[ ${hpss_machines[@]} =~ ${platform,,} ]] ; then
+  source ${workspace}/ush/load_modules_wflow.sh ${SRW_PLATFORM}
+  module load hpss
+  export PYTHONPATH=${workspace}/ush
+fi
+
 [[ ${FORGIVE_CONDA} == true ]] && set +e +u    # Some platforms have incomplete python3 or conda support, but wouldn't necessarily block workflow tests
 conda activate regional_workflow
```

Comment:
As part of the changes in PR #793, the conda environment used with the SRW App is now workflow_tools.  It might be a good idea to update line 96 with:

`conda activate workflow_tools`

```suggestion
conda activate workflow_tools
```

Reply:
okay. new commit.

Reply:
Thanks, @BruceKropp-Raytheon!  Resolving conversation now.